packages/bundle-source/src/bundle-source.js

```diff
@@ -52,4 +51,4 @@ const bundleSource = async (
   }
 };
 
-export default bundleSource;
+export default /** @type {import('./types').BundleSource} */ (bundleSource);
```

Comment:
this seems redundant w.r.t. the type annotation where `bundleSource` is declared.

Reply:
I’ve changed this to `@satisfies` and it seems to work. But, @michaelfig recast the type from `BundleSourceGeneral` to `BundleSource` and that appears to have been necessary. I will confess to not fully understanding the `bundleSource` overloads.